tinygrad/helpers.py

```diff
@@ -3,6 +3,7 @@
 
 def dedup(x): return list(dict.fromkeys(x))   # retains list order
 def prod(x): return math.prod(x)
+def slcfix(i, sz, default): return default if i is None else max(0, min(sz, sz+i if i < 0 else i))
```

Comment:
Same issue, this isn't universal, it's used in one function and shouldn't be in helpers.